Chinese/Hei Hei Hei/Hei Hei Hei.md

```diff
@@ -0,0 +1,7 @@
+够了！！大家都安静点！不要再扯孙悟空了，都过来看费玉污讲段子了！！
+段子在此：
+[如果你能抓到我](Chinese/Kong-Shi-foo/)
+[我就让你](Chinese/Peter/)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.